clog/loggers.py

```diff
@@ -303,7 +303,7 @@ def __init__(self):
         self.lines = {}
 
     def log_line(self, stream, line):
-        assert isinstance(line, six.text_type)
+        assert isinstance(line, (bytes, six.string_types))
```

Comment:
This should be `(bytes, six.text_type)` (string_types includes `bytes` in python2)

Should you alos assert on the stream name too?

Double check what types are legal in python3.  I'm pretty sure it accepts bytes but we wouldn't want to get this wrong :)